static/app/utils/replays/fetchReplayList.tsx

```diff
@@ -26,12 +30,19 @@ type State = {
 
 type Result = State;
 
+type Props = {
+  api: Client;
+  eventView: EventView;
+  location: Location;
+  organization: Organization;
+};
```

Comment:
i was looking at adding an `AbortSignal` prop here, but `api.requestPromise` can't do anything with it, so it seemed easier to keep it in the hook layer, not put anything into this util.

With a signal here or not... the server is still going to do all the work and send data back down the pipe. So it's not worth the messier code.